src/pages/dho/Treasury.vue

```diff
@@ -363,31 +365,31 @@ q-page.page-treasury
-                  img.table-icon(size="10px" v-if="isToken(props.row.amount_requested, 'USD')" src="~assets/icons/husd.png")
-                  img.table-icon(size="10px" v-if="isToken(props.row.amount_requested, 'SEEDS')" src="~assets/icons/seeds.png")
-                  | &nbsp;{{ formatCurrency(props.row.amount_requested) }}
-              q-td(key="requested_date" :props="props")
-                p.q-py-md.q-ma-none.text-italic {{ formatDate(props.row.requested_date) }}
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'HYPHA')" src="~assets/icons/hypha.svg")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'HVOICE')" src="~assets/icons/hvoice.png")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'USD')" src="~assets/icons/husd.png")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'SEEDS')" src="~assets/icons/seeds.png")
+                  | &nbsp;{{ formatCurrency(props.row.amountRequested) }}
+              q-td(key="requestedDate" :props="props")
+                p.q-py-md.q-ma-none.text-italic {{ formatDate(props.row.requestedDate) }}
               q-td(key="amountPaid" :props="props")
                 .row.q-py-md.items-center
-                  img.table-icon(size="10px" v-if="isToken(props.row.amount_requested, 'HYPHA')" src="~assets/icons/hypha.svg")
-                  img.table-icon(size="10px" v-if="isToken(props.row.amount_requested, 'HVOICE')" src="~assets/icons/hvoice.png")
-                  img.table-icon(size="10px" v-if="isToken(props.row.amount_requested, 'USD')" src="~assets/icons/husd.png")
-                  img.table-icon(size="10px" v-if="isToken(props.row.amount_requested, 'SEEDS')" src="~assets/icons/seeds.png")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'HYPHA')" src="~assets/icons/hypha.svg")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'HVOICE')" src="~assets/icons/hvoice.png")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'USD')" src="~assets/icons/husd.png")
+                  img.table-icon(size="10px" v-if="isToken(props.row.amountRequested, 'SEEDS')" src="~assets/icons/seeds.png")
                   | &nbsp;{{ formatCurrency(props.row.amountPaidCurrency) }}
               q-td(key="amountEndorsed" :props="props")
                 span(v-if="props.row.amountPaid === 0") open
-                span(v-if="props.row.amountPaid > 0 && props.row.amountPaid < parseFloat(props.row.amount_requested)") pending
-                div(v-if="props.row.amountPaid === parseFloat(props.row.amount_requested)")
+                span(v-if="props.row.amountPaid > 0 && props.row.amountPaid < getAmount(props.row.amountRequested)") pending
```

Comment:
So basically apart from renames, this is the change - use getAmount instead of parseFloat...